debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -152,9 +153,10 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
 
             SlotCreationResult slotCreatedInfo = null;
             if (snapshotter.shouldStream()) {
-                replicationConnection = createReplicationConnection(this.taskContext,
+                replicationConnection = (PostgresReplicationConnection) createReplicationConnection(this.taskContext,
                         connectorConfig.maxRetries(), connectorConfig.retryDelay());
 
+                LOGGER.info("PID for replication connection: {}", replicationConnection.getBackendPid());
```

Comment:
This replication connection ends up being closed. The one where actual streaming takes place is opened in ChangeEventSourceCoordinator.

Also, it would be best to have a single log line that has info on (connector_name, tserver_ip, walsender_pid) -- this will be easy to grep for to get all occurrences for a single connector from the common connector log